### DISCUSSION TITLE
Videos load on desktop app but not on web app

### DISCUSSION THREAD

**Discussion by UmerFarooqBiomedical** [flet-dev/flet#2791] [Q&A]
Hi,

I am using the following example code: https://flet.dev/docs/controls/video
This code works fine with the desktop app and loads the video but when I load it using web view "ft.app(target=main, view=ft.AppView.WEB_BROWSER)", the video player opens but doesn't load the video files. 

What can be the reason? Your quick guidance will be of great help.

**Comment by ndonkoHenri** [comment]
Please share fill out the issue template if you need help.

**Reply by UmerFarooqBiomedical** [reply]
@ndonkoHenri I have shared the actual issue with you. What kind of template you need to fix the issue in your code? This is a very strange response of labelling this as invalid.

**Reply by ndonkoHenri** [reply]
This discussion has been created from the issue you opened. (Issue-to-discussion conversion)
When opening an issue to report a bug, an issue template is shown, which you are required to fill up.
Without those informations, the issue (bug report) is invalid - and later closed if complementary info is not provided. 

If the issue you wanted to report is still persistent, feel free to open a new issue by clicking [here](https://github.com/flet-dev/flet/issues/new?assignees=&labels=&projects=&template=bug_report.md), then fill in the report details.
When sharing code, please share a runnable code sample so we can test the issue. You could also add screenshots or captures illustrating the issue.

Thanks for your comprehension.
